modules/ROOT/pages/bundling-localization.adoc

```diff
@@ -1,30 +1,27 @@
 = Bundling the User Interface localizations for TinyMCE
 
 :title_nav: UI localizations
-
 :description_short: Information on bundling User Interface localizations
 :description: Information on bundling User Interface localizations
 
 == Overview
 
-* <<usingcommunitylocalizations, Using community localizations>>
-* <<usingpremiumself-hostedlocalizations, Using premium self-hosted localizations>>
+* xref:usingcommunitylocalizations[Using community localizations]
+* xref:usingpremiumself-hostedlocalizations[Using premium self-hosted localizations]
 
+[[usingcommunitylocalizations]]
 == Using community localizations
 
 :forModuleLoaders: true
-
-
-include::partial$misc/using-community-lang-packs.adoc[]
+include::partial$misc/using-community-lang-packs.adoc[leveloffset=+1]
 :forModuleLoaders: false
 
+[[usingpremiumself-hostedlocalizations]]
 == Using premium self-hosted localizations
 
 :editorcomponent: localization
-
-
-include::partial$module-loading/bundling-ref-example.adoc[]
-:editorcomponent: nil
+include::partial$module-loading/bundling-ref-example.adoc[leveloffset=+1]
+:!editorcomponent:
```

Comment:
this is an accident waiting to happen, just have to be mindful to undeclare variables after they are used as it can be picked up by somthing else below this